Sources/Kitura/RouterElement.swift

```diff
@@ -30,7 +30,11 @@ class RouterElement {
 
     /// The regular expression
     #if os(Linux)
-        private var regex: RegularExpression?
+        #if swift(>=3.1)
+            private var regex: NSRegularExpression?
+        #else
+            private var regex: RegularExpression?
+        #endif
```

Comment:
I'm sure it would be. I wasn't trying to determine the intent of the original code, I was just following along with the existing pattern and trying to reduce the possibility of side-effects as much as possible..